### PR TITLE
add LLVM installation instructions for Fedora

### PR DESCRIPTION
**Pull request by CicadaCinema**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,8 +63,16 @@ Jump to [FAQ](#faq).
 ## Installing LLVM
 `package:ffigen` uses LLVM. Install LLVM (9+) in the following way.
 
-#### ubuntu/linux
-1. Install libclangdev - `sudo apt-get install libclang-dev`.
+#### Linux
+1. Install libclangdev.
+
+##### Ubuntu
```
Comment (dcharkes) [inline README.md:69]: Maybe omit the Ubuntu/Fedora headers and just make it:

```
#### Linux

1. Install libclangdev.
   With apt-get: `sudo apt-get install libclang-dev`.
   With dnf: `sudo dnf install clang-devel`.
```

#### Linux

1. Install libclangdev.
   With apt-get: `sudo apt-get install libclang-dev`.
   With dnf: `sudo dnf install clang-devel`.

**Review by dcharkes — CHANGES_REQUESTED** [review]
Thanks @CicadaCinema !

**Comment by CicadaCinema** [comment]
done!

**Review by dcharkes — APPROVED** [review]
tyvm @CicadaCinema !

**Pull request merged**